code/modules/mob/living/basic/pets/cat/cat_house.dm

```diff
@@ -0,0 +1,46 @@
+/obj/structure/cat_house
```

Comment:
should this remain in the `/mob/basic` directory or should we move it to somewhere in the `/obj/structure` directory? doesn't seem uesful here